internal/node/launcher.sh

```diff
@@ -188,9 +188,14 @@ ARGS=()
 LAUNCHER_NODE_OPTIONS=( "--require" "$require_patch_script" )
 USER_NODE_OPTIONS=()
 ALL_ARGS=(TEMPLATED_args $NODE_REPOSITORY_ARGS "$@")
+STDOUT_CAPTURE=/dev/stdout
+STDERR_CAPTURE=/dev/stderr
```

Comment:
I'm unsure about this, as when a nodejs_binary is run, it will emit error to stderr, but when npm_package_bin is run and produces it currently swallows them, this would therefore force them all to stderr, I'm not sure if this is intentional